src/frontend/packages/lib_components/src/utils/retryQuery.ts

```diff
@@ -8,8 +6,11 @@ type ShouldRetryFunction<TError> = (
 ) => boolean;
 
 export const retryQuery: ShouldRetryFunction<unknown> = (
-  _failureCount: number,
+  failureCount: number,
   error,
 ) => {
-  return (error as FetchResponseError).status !== 403;
+  if ((error as FetchResponseError).status === 403) {
+    return false;
+  }
+  return failureCount < 3;
```

Comment:
`failureCount` starts at `0` or `1` ?

Reply:
`failureCount` starts as 0.
So 4 requests max are made (like with default options): 
- The initial request
- 3 retries